1209/345C/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Keyboard Controller
+owner: kinX
+license: CERN Open Hardware Licence v1.2
+site: https://michael.stapelberg.ch/posts/2018-04-17-kinx/
+source: https://github.com/kinx-project/mk66f-hw
+---
```

Comment:
Can you please link to the firmware source in the body of the file too?

Reply:
Sorry, not sure I follow. Your comment is literally one line above a link to the firmware in the body of the file, so maybe I misunderstand what you mean? If you meant to comment on the other file, there is no firmware required for the USB hub IC which that project uses, but its parameters are customizable, so a custom VID/PID still makes sense.

Thanks,

Reply:
Sorry, I managed to miss that somehow.